products/wso2_EI/business_process_orchestration/HumanTasksSamples/EmailNotificationSample/README.md

```diff
@@ -0,0 +1,113 @@
+# README
+**Author :** Dilshani Subasinghe
+
+### Description
+
+The BPEL specification defines human tasks for the business processes that need human interactions in order to make decisions. These human tasks can generate notifications to inform the users of the various decisions that need to be made. Certain users were able to view the notifications from their account. However, with the email and SMS notification support, users can configure the BPS to send messages to the specified email addresses when a notification is created. So this sample created to test particular use case
+
+This folder contains sample for BPS Human task email notifications which will trigger mail due to a defined deadline.
+ * ClaimsApprovalTask.ht
+ * ClaimsApprovalTaskService.wsdl
+ * htconfig.xml
+ * ApproveClaim-input.jsp
+ * ApproveClaim-output.jsp
+ * ApproveClaim-response.jsp
+ * ClaimApprovalReminder-input.jsp
+
+Note: Create zip file using taskDeadlineWithNotifications folder
+
+### Setup 
+
+    Product Version: WS02 BPS 3.6.0, WSO2 EI 6.2.0
+
+### Contents
+
+  - ClaimsApprovalTask.ht
+  - ClaimsApprovalTaskService.wsdl
+ 
+Note: Deploy ClaimReminderService and ClaimService
+    
+### Pre-conditions
+
+- Enable email notifications in BPS. 
+Refer: https://docs.wso2.com/display/BPS360/Enabling+Email+and+SMS+Notifications+for+Human+Tasks
+
+### Execution steps
+1. Start business process profile 
+2. Login in to carbon console (Ex: https://localhost:9443/carbon/)
+3. Create roles as "regionalClerks","regionalManager" and "clerksManager"

```

Comment:
According to the given ClaimsApprovalTask.ht, The roles that need to be created are "regionalManagerRole" and "regionalClerksRole". If these roles are not defined human task deployment get failed.

Reply:
Noted, Updated.